scripts/plugins/test_extauth_hook_AD.py

```diff
@@ -144,9 +144,18 @@ def test_not_permit_non_admin_group(self, mock_rename, mock_chmod):
 
     @patch("extauth_hook_ad.run_cmd")
     def test_query_subject_success(self, mock_cmd, mock_rename, mock_chmod):
+        def cmd_result(cmd):
```

Comment:
I find the name confusing. Is this not simulating the execution of a command? Something like `mock` or `mock_exec` would tell me more.